GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -72,10 +86,13 @@ private bool TryInitializeUpgrader(out string error)
                     jsonTracer.AddLogFileEventListener(logFilePath, EventLevel.Informational, Keywords.Any);
 
                     this.tracer = jsonTracer;
-                    this.prerunChecker = new InstallerPreRunChecker(this.tracer, this.Confirmed ? GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm : GVFSConstants.UpgradeVerbMessages.GVFSUpgrade);
+                    this.prerunChecker = new InstallerPreRunChecker(

```

Comment:
As discussed offline, display an error (and fail the `gvfs upgrade`) if both `--dry-run` and `--confirm` are passed in.

Reply:
Added in `UpgradeVerb.cs` line no. 70